db/migrate/20220824121504_add_cookies_consent_ui_to_subdomain.rb

```diff
@@ -1,12 +1,16 @@
 class AddCookiesConsentUiToSubdomain < ActiveRecord::Migration[6.1]
   COOKIE_CONSENT_UI = 
   "<div class=\"cookies-consent\">
-    <h2 class=\"cookies-consent__title\">We Value Your Privacy</h2>
-    <p>
-      We use cookies to enhance your browsing experience, serve personalized ads or content, and analyze our traffic. By clicking \"Accept All\", you consent to our use of cookies.
-    </p>
-    <a class=\"btn btn-primary mr-2 mb-2 mb-sm-0\" href=\"/cookies?cookies=true\">Accept All</a>
-    <a class=\"btn btn-outline-primary mb-2 mb-sm-0\" href=\"/cookies?cookies=false\">Reject All</a>
+    <div class=\"cookies-consent__text-content\">
```

Comment:
@Ayon95 [per my comment here](https://github.com/restarone/violet_rails/pull/1070/files#r956800732)-- we should ideally only change the default migration value

Reply:
Isn't this the default migration value?

Reply:
@Ayon95 yup exactly. I was referring to not making the change here (`app/views/cookies/index.html.erb`)

Reply:
Oh okay. I can remove the overlay from `app/views/cookies/index.html.erb` and include it in the default markup.